tests/features/person.feature

```diff
@@ -81,25 +81,31 @@ Feature: Person content creation
-    And I fill in "Email" with "person_contact@example.com" in the "Person contact" region
-    And I fill in "Phone number" with "0488779033" in the "Person contact" region
-    And I fill in "Mobile number" with "0488779034" in the "Person contact" region
-    And I fill in "Fax number" with "0488779035" in the "Person contact" region
-    And I select "Hungary" from "Country" in the "Person contact" region
+    And I fill in "Name" with "Name of the contact" in the "Person contacts" region
+    And I fill in "Organisation" with "Person contact organisation" in the "Person contacts" region
+    And I fill in "Body text" with "Person contact body text" in the "Person contacts" region
+    And I fill in "Website" with "http://www.example.com/person_contact" in the "Person contacts" region
+    And I fill in "Email" with "person_contact@example.com" in the "Person contacts" region
+    And I fill in "Phone number" with "0488779033" in the "Person contacts" region
+    And I fill in "Mobile number" with "0488779034" in the "Person contacts" region
+    And I fill in "Fax number" with "0488779035" in the "Person contacts" region
+    And I select "Hungary" from "Country" in the "Person contacts" region
     And I wait for AJAX to finish
-    And I fill in "Street address" with "Person contact street" in the "Person contact" region
-    And I fill in "Postal code" with "9000" in the "Person contact" region
-    And I fill in "City" with "Budapest" in the "Person contact" region
-    And I fill in "Office" with "Person contact office" in the "Person contact" region
+    And I fill in "Street address" with "Person contact street" in the "Person contacts" region
+    And I fill in "Postal code" with "9000" in the "Person contacts" region
+    And I fill in "City" with "Budapest" in the "Person contacts" region
+    And I fill in "Office" with "Person contact office" in the "Person contacts" region
     And I fill in "URL" with "mailto:person_contact_social@example.com" in the "Contact social media links" region
     And I fill in "Link text" with "Person contact social link email" in the "Contact social media links" region
-    And I fill in "Media item" with "Contact image" in the "Person contact" region
-    And I fill in "Caption" with "Person contact caption" in the "Person contact" region
-    And I fill in "Press contacts" with "http://example.com/press_contacts" in the "Person contact" region
+    And I fill in "Media item" with "Contact image" in the "Person contacts" region
+    And I fill in "Caption" with "Person contact caption" in the "Person contacts" region
+    And I fill in "Press contacts" with "http://example.com/press_contacts" in the "Person contacts" region
+    # Jobs field.
+    And I press "Add new person job"
+    And I wait for AJAX to finish
+    And I fill in "first" person job role reference field with "Asia-Pacific Economic Cooperation"
```

Comment:
Role term from triple store has to be used. It is temporary solution just for testing.